services/121-service/src/payments/services/fsp-env-variable-validation.service.ts

```diff
@@ -0,0 +1,59 @@
+import { Injectable, Logger } from '@nestjs/common';
+
+import { Fsps } from '@121-service/src/fsp-management/enums/fsp-name.enum';
+import { FspEnvVariablesDto } from '@121-service/src/fsp-management/fsp-env-variables.dto';
+
+@Injectable()
+export class FspEnvVariableValidationService {
+  private readonly logger = new Logger(FspEnvVariableValidationService.name);
+
+  // A pure function.
```

Comment:
what do you mean with this? My understanding is that a function that is part of a class, is a method. 

Also when you don't define private or public, I though the default was private. But if so, then how can you call it in the spec file? 

Please enlighten me :)

Reply:
Jannis abides.